vdc/k8s-config.yaml

```diff
@@ -144,3 +204,14 @@ roleRef:
   kind: Role
   name: test-test
   apiGroup: "rbac.authorization.k8s.io"
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
```

Comment:
I was under the impression we needed to declare a persistent volume for the claim to "claim". Is that declared somewhere or am I mistaken?

Reply:
It is created automatically.